src/mixins/dom.js

```diff
@@ -0,0 +1,44 @@
+// DomMixin
+//  ---------
+
+import Backbone from 'backbone';
+
+export default {
+  createBuffer() {
+    return document.createDocumentFragment();
+  },
+
+  appendChildren(el, children) {
+    Backbone.$(el).append(children);
+  },
+
+  beforeEl(el, sibling) {
+    Backbone.$(el).before(sibling);
+  },
+
+  replaceEl(newEl, oldEl) {
+    const parent = oldEl.parentNode;
+
+    if (!parent) {
+      return;
+    }
+
+    parent.replaceChild(newEl, oldEl);
+  },
+
+  detachContents(el) {
+    Backbone.$(el).contents().detach();
+  },
+
+  attachHTML(el, html) {
```

Comment:
re: @blikblum comment below this could be `attachContents` to align with `detachContents`.. it's not quite the same, but they are the opposite functions.